src/net/sparkzz/modest/io/config/YAMLConfig.java

```diff
@@ -84,9 +84,7 @@ else if (((Number) tempObject).intValue() == 0)
 	public boolean hasValue(String key) {
 		tempObject = data.get(key);
 
-		if (tempObject != null)
-			return true;
-		return false;
+		return (tempObject != null);
```

Comment:
> Is it more efficient to have a single object that just gets overwritten every time it's called or to just create a new method?

That is a good question, never really thought about it... I'll have to look into that when I have the time....